junit-platform-commons/src/main/java/org/junit/platform/commons/util/ClasspathScanner.java

```diff
@@ -102,7 +102,7 @@
 			Path baseDir = closeablePath.getPath();
 			return findClassesForPath(baseDir, basePackageName, classFilter);
 		}
-		catch (PreconditionViolationException ex) {
+		catch (org.junit.platform.commons.PreconditionViolationException ex) {
```

Comment:
❓ Can we replace this with an import as well?

Reply:
Done.